html/comparison.html.haml

```diff
@@ -17,640 +15,11 @@
-            %td {{service.name}}
-            %td{:class=>"{{object.init_system}}_{{service.state}}"}
-              {{service.state}}
-    %script#scope_services.partial{:type => "text/ng-template"}
-      %a.scope_anchor{:id => "services"}
-      #services_container.scope
-        .row
-          .col-xs-1
-            %a{:href => "#services"}
-              %img.scope_logo_big{:src => "/assets/logo-services.png", :class=>"over", :"data-toggle"=>"popover", :title=>"Services", :"data-content"=>"#{scope_help('services')}"}/
-            %span.toggle{:title => "Collapse/Expand"}
-          .col-xs-11
-            %h2
-              Services
-              .scope-summary
-                %span.summary-part{"ng-show" => "diff.services.only_in1"}
-                  {{diff.meta.description_a}}: {{diff.services.only_in1.services.length || 0}} services ({{diff.services.only_in1.init_system}})
-                %span.summary-part{"ng-show" => "diff.services.only_in2"}
-                  {{diff.meta.description_b}}: {{diff.services.only_in2.services.length || 0}} services ({{diff.services.only_in2.init_system}})
-                %span.summary-part{"ng-show" => "diff.services.common"}
-                  %a{:class=>"show-common-elements", href: "#services_both"}<
-                    both
-                  = ": {{diff.services.common.services.length}} services ({{diff.services.common.init_system}})"
-        .row.scope_content.collapse.in
-          .row
-            .col-xs-1
-            .col-xs-5{"ng-show" => "diff.services.only_in1"}
-              %only-in-a
-              %render-template{:template => "scope_services_partial", :object => "diff.services.only_in1"}
-            .col-xs-1
-            .col-xs-5{"ng-show" => "diff.services.only_in2"}
-              %only-in-b
-              %render-template{:template => "scope_services_partial", :object => "diff.services.only_in2"}
-          %a.both_anchor{id: "services_both"}
-          .row
-            .col-xs-1
-            .col-xs-2
-              %a{:class=>"show-common-elements", :href => "#", "ng-show" => "diff.services.common"}
-                Show common elements in services
-              %a{:class=>"hide-common-elements", :href => "#", :style => "display: none", "ng-show" => "diff.services.common"}
-                Hide common elements in services
-          .row.scope_common_content.collapse{"ng-show" => "diff.services.common"}
-            .col-xs-1
-            .col-xs-11
-              %in-both
-              %render-template{:template => "scope_services_partial", :object => "diff.services.common"}
-
     %script{:src => "/assets/compare/machinery.js"}
     %script{:src => "/assets/bootstrap-tooltip.js"}
     %script{:src => "/assets/bootstrap-popover.js"}
 
-  %body{"ng-controller" => "compareController", "data-description-a" => description_a, "data-description-b" => description_b}
+  %body{"data-description-a" => @description_a.name, "data-description-b" => @description_b.name}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace